### PR TITLE
removed column fetch_offset_timezone

### PR DESCRIPTION
**Pull request by Shrhawk**
**Checklist**:

* [ ] All tests are passing
* [ ] I have added any necessary documentation and/or release notes
* [ ] I have followed the [commit message guidelines](https://wiki.antonischristofides.com/en/sops/production/development/git-and-github#committing-and-commit-messages) and the ["before submitting a pull request" guidelines](https://wiki.antonischristofides.com/en/sops/production/development/git-and-github#before-submitting-a-pull-request)
* [ ] If I introduced (or abolished) any third-party library, I will write a short paragraph in this PR [according to the guidelines](https://wiki.antonischristofides.com/sops/production/development/third-party-libraries)
* [ ] I have removed all obsoleted code

### CODE REVIEW

**Review by aptiko — CHANGES_REQUESTED** [review]
What was the problem with migrations? Doesn't `./manage.py makemigrations enhydris` do it?

**Comment by Shrhawk** [comment]
> What was the problem with migrations? Doesn't `./manage.py makemigrations enhydris` do it?

Django is not detecting any model change when I run the command. I also have tried without app name but then I get file write permission denied error.

File: enhydris/telemetry/tests/test_models.py [hunk 1]
```diff
@@ -72,11 +71,10 @@ class TelemetryIsDueTestCase(TestCase):
     def setUpClass(cls):
         super().setUpClass()
         cls.telemetry = Telemetry(
-            station=mommy.make(Station),
+            station=Station.objects.create(),
```
Comment (aptiko) [inline enhydris/telemetry/tests/test_models.py:74]: There's no need to change this, and, in fact, it won't work.

**Review by aptiko — CHANGES_REQUESTED** [review]
I attach the migrations file. However, you should be able to create it yourself. I did nothing but run `./manage.py makemigrations`. Are you on Windows? There are some tricky issues with permissions (uid mapping) on Windows, it might be related, but this is guesswork. If `./manage.py makemigrations enhydris` claims there are no changes, then the most likely reasons are that you are on the wrong branch or that you do not have `enhydris.telemetry` in the settings.
[0008.zip](https://github.com/openmeteo/enhydris/files/12479190/0008.zip)

Some tests are failing, but you'll find this when you try to run them.

**Comment by aptiko** [comment]
By the way, the commit message should also be `Remove column fetch_offset_timezone (fixes #507)` as described in "Git and GitHub" in confluence.

**Comment by aptiko** [comment]
Merged in 9211d7fd68698fbf6d8ca0024eaefbfcc321114e.

**Pull request closed without merging**